database/migrations/2021_02_10_100000_add_checkout_permissions.php

```diff
@@ -8,29 +8,22 @@ class AddCheckoutPermissions extends BasePermissionsMigration
 {
     public function up()
     {
+        $permissions = [
+            'view orders',
+            'view order items',
+        ];
+        $this->createPermissions($permissions);
```

Comment:
I like leaving this how it is so that it is clear that this not only adds all the package permissions to the database table but also assigns them to the admin role.